translator/source_code/icone_translator.py

```diff
@@ -299,14 +319,16 @@ def parse_incident(incident, callback_function=None):
     properties['ending_accuracy'] = "estimated"
 
     # road_name
-    road_name = incident.get('location').get('street')
-    properties['road_name'] = road_name
+    road_names = [incident.get('location').get('street')]
+    properties['road_names'] = road_names
 
     # direction
-    direction = parse_direction_from_street_name(road_name)
+    direction = parse_direction_from_street_name(road_names[0])
```

Comment:
If we have multiple road names in this array, are we guaranteed to be able to get the direction from the first name? We may want to update the `parse_direction_from_street_name` method to handle the full array and loop through to find the proper direction. Or alternatively, loop through here to find a valid direction (or end of array) before moving on.

Reply:
should be fixed